[TW-1556] Create overview for collaborating in postman (#5379)
postmanlabs/postman-docs: src/pages/docs/collaborating-in-postman/working-with-your-team/team-collaboration.md

    url: "https://www.postman.com/case-studies/paypal/"
---

> Postman Free lets teams of up to three work together at no cost. To collaborate with more team members, features, and increased usage limits, see [Plans and Pricing](https://www.postman.com/pricing/).

You can create and join teams in Postman, enabling you to collaborate with teammates in your organization. You can also choose to keep workspaces in your Postman account separate from teams you join. You can be a member of up to ten Postman teams, regardless of their [plan type](https://www.postman.com/pricing/).

> If an Enterprise team within your organization implements [domain capture](/docs/administration/domain-verification-and-capture/domain-capture-overview/), you won't be able to remain on or join extra Postman teams with your captured accounts.

## Contents

* [About accounts and teams](#about-accounts-and-teams)
* [Creating a team](#creating-a-team)
* [Joining a team](#joining-a-team)
* [Switching between teams](#switching-between-teams)
* [Leaving a team](#leaving-a-team)
* [Team usage limits](#team-usage-limits)
* [Next steps](#next-steps)

## About accounts and teams

> When you sign up for Postman as an individual, you create a [Postman account](/docs/getting-started/installation/postman-account/). If you [join a Postman team](#joining-a-team), you can decide to either move your workspaces and transfer ownership to the team or keep your workspaces separate and maintain ownership of them in your account. This decision affects your account:

Reviewer: ```suggestion
When you sign up for Postman as an individual user, you create a [Postman account](/docs/getting-started/installation/postman-account/). If you [join a Postman team](#joining-a-team), you can either move your workspaces and transfer ownership to the team or keep your workspaces separate and maintain ownership in your account. This decision affects your account in the following ways:
```

## Leaving a team

You can leave a Postman team by selecting your avatar in the top right, then **Settings**. Select **Teams** on the left.

<img src="https://assets.postman.com/postman-docs/dashboard-teams-leave-team-v9.2.jpg" alt="Leave team"/>

Select **Leave Team** to the right of a team to leave it. If you're a member of an Enterprise team that has [SCIM configured](/docs/administration/scim-provisioning/scim-provisioning-overview/), you must contact a Team Admin to remove you from the team.

When you leave a team, you no longer have access to the team's workspaces, including personal workspaces, or any of the elements in them. You must reassign your personal workspaces to a remaining team member before leaving if one of the following is true:

* You're from an Enterprise team.
* You're from a Free, Basic, or Professional team, and you have an account.

> An account isn't associated with a team, and only you can access it. To access your account, see [Switching between teams](#switching-between-teams).

<img src="https://assets.postman.com/postman-docs/v10/leave-team-confirmation-v10-2.jpg" alt="Leave and delete team" width="500px"/>

> When you leave a team, you can also choose to keep your personal workspaces and the data within them in an account if the following is true: you're from a Free, Basic, or Professional team and you don't have an account.

Reviewer: ```suggestion
When you leave a team, you can also choose to keep your personal workspaces and the data within them in an account if you're from a Free, Basic, or Professional team and you don't have an account.
```

## Team usage limits

Your team's usage limits are based on your [plan type](https://www.postman.com/pricing/) and any [add-ons](/docs/billing/billing/#purchasing-add-on-resources) your team has purchased.

Open the [**Resource usage** dashboard](http://go.postman.co/billing/add-ons/overview) to see your team's usage of mock servers and monitors, APIs created, collection runner runs, image and file storage, Flow requests, performance tests, integrations, and the Postman Cloud Agent. To open the **Resource usage** dashboard, do one of the following:

* **Free users** - In the Postman header, select the menu next to **Upgrade**.
* **Paid users** - In the Postman header, select **Team**.

> To learn more about the resources included with your Postman plan and what happens when you reach your usage limits, go to [About resource usage](/docs/billing/resource-usage/).

> <img src="https://assets.postman.com/postman-docs/team-dropdown-resource-usage-v9-25.jpg" alt="Team resource usage information" width="342px"/>

Reviewer: This image could use an update

Author: This is now updated. Thanks!
